templates/comments/list.html

```diff
@@ -141,4 +141,10 @@ <h3>{{ _('New comment') }}</h3>
             {% endif %}
         </div>
     {% endif %}
+
+    {% if comment_lock %}
+        <div style="width: 100%; padding-left: 14px" class="alert alert-warning">
```

Comment:
I would rather not inline this style, we have a comment SCSS for things like this.